src/bitmanip.h

```diff
@@ -5,7 +5,7 @@
 #ifndef BITCOIN_UTIL_BITMANIP_H
 #define BITCOIN_UTIL_BITMANIP_H
 
-#include <config/bitcoin-config.h>
+#include "config/bitcoin-config.h"
```

Comment:
what was the compilation error issued by this one?

Reply:
the android compiler wasn't finding bitcoin-config.h.  However, there was another problem causing this, so maybe the android compiler would have worked with <>.  But <> is not technically correct -- they tell the compiler to look in the system directories not the local directories.  However, I think most modern compilers have given up on this distinction.  But for consistency's sake we should do it properly:

$ grep -R "bitcoin-config.h" *
bitcoin-cli.cpp:#include "config/bitcoin-config.h"
bitcoind.cpp:#include "config/bitcoin-config.h"
bitcoin-miner.cpp:#include "config/bitcoin-config.h"
bitcoin-tx.cpp:#include "config/bitcoin-config.h"
bitmanip.h:#include <config/bitcoin-config.h>
clientversion.h:#include "config/bitcoin-config.h"
compat/glibc_sanity.cpp:#include "config/bitcoin-config.h"
compat/strnlen.cpp:#include "config/bitcoin-config.h"
compat/endian.h:#include "config/bitcoin-config.h"
compat/byteswap.h:#include "config/bitcoin-config.h"
compat/glibc_compat.cpp:#include "config/bitcoin-config.h"
compat.h:#include "config/bitcoin-config.h"
config/bitcoin-config.h.in:/* src/config/bitcoin-config.h.in.  Generated from configure.ac by autoheader.  */
crypto/common.h:#include "bitcoin-config.h"
init.cpp:#include "config/bitcoin-config.h"
leakybucket.h:#include "config/bitcoin-config.h"
main.h:#include "config/bitcoin-config.h"
Makefile.in:CONFIG_HEADER = $(top_builddir)/src/config/bitcoin-config.h
netaddress.h:#include "config/bitcoin-config.h"
net.cpp:#include "config/bitcoin-config.h"
qt/optionsdialog.cpp:#include "config/bitcoin-config.h"
qt/receiverequestdialog.cpp:#include "config/bitcoin-config.h" /* for USE_QRCODE */
qt/intro.cpp:#include "config/bitcoin-config.h"
qt/utilitydialog.cpp:#include "config/bitcoin-config.h"
qt/rpcconsole.cpp:#include "config/bitcoin-config.h"
qt/splashscreen.cpp:#include "config/bitcoin-config.h"
qt/askpassphrasedialog.cpp:#include "config/bitcoin-config.h"
qt/unlimiteddialog.cpp:#include "config/bitcoin-config.h"
qt/bitcoingui.h:#include "config/bitcoin-config.h"
qt/addressbookpage.cpp:#include "config/bitcoin-config.h"
qt/optionsmodel.cpp:#include "config/bitcoin-config.h"
qt/test/test_main.cpp:#include "config/bitcoin-config.h"
qt/bitcoin.cpp:#include "config/bitcoin-config.h"
qt/bitcoingui.cpp:#include "config/bitcoin-config.h"
qt/notificator.h:#include "config/bitcoin-config.h"
qt/unlimitedmodel.cpp:#include "config/bitcoin-config.h"
script/bitcoinconsensus.h:#include "config/bitcoin-config.h"
support/pagelocker.cpp:#include "config/bitcoin-config.h"
test/test_bitcoin_fuzzy.cpp:#include "config/bitcoin-config.h"
timedata.cpp:#include "config/bitcoin-config.h"
util.cpp:#include "config/bitcoin-config.h"
util.h:#include "config/bitcoin-config.h"
utiltime.cpp:#include "config/bitcoin-config.h"
zmq/zmqconfig.h:#include "config/bitcoin-config.h"

Reply:
thanks for the explanation, I was asking just because I saw tendency in other C++ project to move from `""` to `<>` even for non system headers